src/styles/main.scss

```diff
@@ -53,16 +56,18 @@ body {
     border: 1px solid #000;
     background-color: #eee;
     cursor: pointer;
-    transition: background-color 0.5s, transform 0.5s;
+    transition-duration: 0.5s;
```

Comment:
don't animate all properties, just that you need, move `0.5s` into variable if you repeat it

Reply:
1) This was modified last time:
https://github.com/Kasapulka/layout_calendar/blob/develop/src/styles/main.scss
2) The task was:
Implement start-day modifier for the calendar with mon, tue, wed, thu, fri, sat and sun values
Use [@each] to create all the modifiers
I don`t understand what is wrong.

Reply:
```
$transition-duration: 0.5s;
transition: background-color $transition-duration, transform $transition-duration;
```

Reply:
wasn't fixed